MaterialLib/TwoPhaseModels/CreateTwoPhaseFlowMaterialProperties.cpp

```diff
@@ -24,75 +26,114 @@ namespace MaterialLib
 {
 namespace TwoPhaseFlowWithPP
 {
-std::unique_ptr<TwoPhaseFlowWithPPMaterialProperties>
-CreateTwoPhaseFlowMaterialProperties(
+std::tuple<std::unique_ptr<TwoPhaseFlowWithPPMaterialProperties>,
+           BaseLib::ConfigTree>
+createTwoPhaseFlowMaterialProperties(
     BaseLib::ConfigTree const& config,
-    bool const has_material_ids,
-    MeshLib::PropertyVector<int> const& material_ids)
+    boost::optional<MeshLib::PropertyVector<int> const&>
+        material_ids)
 {
     DBUG("Reading material properties of two-phase flow process.");
 
-    //! \ogs_file_param{prj__processes__process__TWOPHASE_FLOW_PP__material_property__fluid}
-    auto const& fluid_config = config.getConfigSubtree("fluid");
+    //! \ogs_file_param{material__twophase_flow_pp__material_property__fluid}
```

Comment:
and change it to:
```
\ogs_file_param{prj__processes__process__TWOPHASE_FLOW__material_property__porous_medium__porous_medium__capillary_pressure}
```
Besides, this class should be move to ProcessLib/TwoPhaseFlow. This could be done in the next PR.